### PR TITLE
refs task commit test conflict

### PR DESCRIPTION
**Pull request by huuhv**
Please review

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ abcde
 abcdef
 abcdefg
 123456789
+test conflict
```
Comment (vuhuytuan89) [inline test.txt:6]: asdasdasd

File: test.txt [hunk 2]
```diff
@@ -3,4 +3,5 @@ abcde
 abcdef
 abcdefg
```
Comment (vuhuytuan89) [inline test.txt:4]: sASAsASA

**Review by vuhuytuan89 — COMMENTED** [review]
fix 2

File: test.txt (hunk 2 above)
Comment (huuhv) [reply test.txt:4]: cai nay ko can fix dau, vi

**Pull request merged**